src/game/CycleGame.ts

```diff
@@ -13,8 +13,8 @@ import { Languages } from './states/languages';
 export class CycleGame {
     private game: Phaser.Game;
 
-    constructor() {
-        this.game = new Phaser.Game(BASE_SIZE, BASE_SIZE, Phaser.CANVAS, 'cycleCanvas');
+    constructor(private readonly htmlElement: string, public readonly baseSize = BASE_SIZE) {
```

Comment:
`public readonly baseSize = BASE_SIZE` why not `private`?

Reply:
It needs to be public to be able to set the height and width of the HTML element:
https://github.com/cycle-game/Cycle/pull/39/files#diff-34456421648850188ad56fcd6df47b2bR21